modular_zubbers/code/datums/id_trim/jobs.dm

```diff
@@ -24,6 +24,11 @@
 	job = /datum/job/blacksmith
 
 /datum/id_trim/job/blueshield
+	extra_access = list(ACCESS_COURT, ACCESS_CARGO, ACCESS_GATEWAY) //Blueshield loses Brig and Security access and cannot perform arrests.
```

Comment:
Yeah but the likelihood this will be abused is almost none, I think we can safely leave it as-is. 
At that point the BS can just get a welder and weld them into a locker.